DevoxxClientMobile/src/main/java/com/devoxx/views/ConfSelectorPresenter.java

```diff
@@ -26,214 +26,115 @@
-        if (conferences.isInitialized()) {
-            selectSavedConference(conferences, configuredConference);
-        } else {
-            selectSavedConferenceListener = o -> selectSavedConference(conferences, configuredConference);
-            conferences.initializedProperty().addListener(selectSavedConferenceListener);
-        }
-    }
-
-    private void selectSavedConference(GluonObservableList<Conference> conferences, String configuredConference) {
-        for (Conference conference : conferences) {
-            if (conference.getId().equals(configuredConference)) {
-                selector.setSelectedItem(conference);
-                break;
+        final Button filter = MaterialDesignIcon.FILTER_LIST.button();
+        MenuPopupView popup = getMenuPopupView(filter);
+        
+        filter.setOnAction(e -> {
+            if (popup.isShowing()) {
+                popup.hide();
+            } else {
+                popup.show();
             }
-        }
-        if (selectSavedConferenceListener != null) {
-            conferences.initializedProperty().removeListener(selectSavedConferenceListener);
-        }
-    }
+            e.consume();
+        });
 
-    private void selectNearestConferenceFrom(GluonObservableList<Conference> conferences) {
-        if (conferences.isInitialized()) {
-            selectNearestConference(conferences);
-        } else {
-            selectNearestConferenceListener = o -> selectNearestConference(conferences);
-            conferences.initializedProperty().addListener(selectNearestConferenceListener);
-        }
+        filterPane.getChildren().add(filter);
+        
     }
 
-    private void selectNearestConference(GluonObservableList<Conference> conferences) {
-        if (conferences.size() == 0) return;
-        long minPositiveDaysUntil = Long.MAX_VALUE;
-        long minNegativeDaysUntil = Long.MIN_VALUE;
-        Conference nearestFuture = null;
-        Conference nearestPast = conferences.get(0);
-        for (Conference conference : conferences) {
-            long days = conference.getDaysUntilEnd();
-            if (days >= 0) {
-                if (days < minPositiveDaysUntil) {
-                    minPositiveDaysUntil = days;
-                    nearestFuture = conference;
-                }
-            } else {
-                if (days > minNegativeDaysUntil) {
-                    minNegativeDaysUntil = days;
-                    nearestPast = conference;
-                }
-            }
-        }
-        if (nearestFuture != null) {
-            selector.setSelectedItem(nearestFuture);
-        } else {
-            selector.setSelectedItem(nearestPast);
-        }
-        if (selectNearestConferenceListener != null) {
-            conferences.initializedProperty().removeListener(selectNearestConferenceListener);
-        }
+    private MenuPopupView getMenuPopupView(Button filter) {
+        Menu menu = new Menu();
```

Comment:
The menu for switching between the conference type is currently unstyled and the labels are only showing the first letters.
![devoxx-conference-type-selector](https://user-images.githubusercontent.com/4688201/44774613-d81a6f80-ab73-11e8-9277-e4a011c819ce.png)

Reply:
This has been fixed in Gluon Mobile. We need a new release of GM and make sure Devoxx app depends on it.